development/docs/write_cli_docs.py

```diff
@@ -1,30 +1,34 @@
 import subprocess
 import os 
 
-FILENAME = os.path.abspath(
+DOCS_ROOT_DIR = os.path.abspath(
     os.path.join(
         os.path.dirname(__file__),
         "..",
         "..",
         "docs",
-        "inference_helpers",
-        "cli_commands",
-        "reference.md"
     )
 )
 
-def write_file(path: str, content: str) -> None:
```

Comment:
Is this refactor intended to not create directories?

Reply:
directory already exists in repo.   this wasn't the root cause, but I tried a million things before figuring out the typer issue.  this is more inline with how all the other scripts do it so decided to leave it